luigi/execution_summary.py

```diff
@@ -89,34 +91,37 @@ def _get_str(task_dict, extra_indent):
     """
     This returns a string for each status
     """
+    summary_length = configuration.get_config().getint('execution_summary', 'summary-length', 5)
```

Comment:
This is still blocking merge. Please do it the luigi way as in the docs I linked to